src/eko/output/struct.py

```diff
@@ -41,7 +53,6 @@ class DictLike:
 
     Some collections and scalar objects are normalized to native Python
     structures, in order to simplify the on-disk representation.
-
```

Comment:
The extra empty line is part of the NumPy style, please, don't remove it:
https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html

Reply:
Looking there: `function_with_pep484_type_annotations` has it indeed, but `function_with_types_in_docstring` not, so I'd say there is no clear indication ... however, in the future I can do, but I'll not revert here

Reply:
Yes, I know, but it is not only ``, there are also:
1. `module_level_function`
2. `example_generator`
3. `ExampleError` class
4. `ExampleClass` class
5. its `__init__` method
6. its `example_method`
7. its `__special__` method
8. and `_private` as well
9. as the module level docstring

I'd say the indication is not so unclear

Reply:
In particular, since you took the burden of removing 8 of them, not a single one, please revert them as well.

If it was only for one, we could have done at any stage, but to enforce a convention we can not keep having plenty of exceptions, including in the new code.

Reply:
Btw: they are also polluting the diff, making it more difficult to assess the status of this PR from the "Files changed" tab. Since this PR is not finishing today, and we'll have to merge and reassess its status, please keep it as clean as possible.